### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by markinruc** [mega-sam/mega-sam#10]
Very great work!

Will the training code or dataset be open source? Or can you provide some guidance?

Thanks a lot!

**Comment by raven38** [comment]
Will the generation code for 11K dynamic videos from Kubric be open-source? Or can we easily adapt from the Kubric examples?

**Comment by holynski** [comment]
The Kubric rendering code won't be released, but we're happy to provide any details that might be missing from the paper.

**Comment by zhengqili** [comment]
Yes. We don't plan to release training and data generation code. but we are happy to provide any details for reproducing the trained model from scratch.

**Comment by raven38** [comment]
In MegaSAM, movement maps are used to train the movement map predictor, but how are movement maps obtained when rendering dynamic videos from Kubric? Kubric provides segmentation maps, which allow identification of foreground and background, but I am concerned that using segmentation maps would result in inaccurate movement maps because both static and dynamic objects exist in the foreground. How can movement maps be obtained?

**Comment by zhengqili** [comment]
We derive ground truth 3D track from Kubric and create mask from moving 3D track

**Comment by xuhaozheng** [comment]
Hi, Thank you for your great work. I wonder whether you use the original Kubric dataset (or any subset), or you created your own dataset from using their render? Specifically, how you process the camera pose and optical flow?
